docs/src/adoc/index.adoc

```diff
@@ -4046,6 +4046,10 @@ Then configure the Jdbi factory bean in your Spring container, e.g.:
     `@Inject` annotations on the target class instead of configuring it in
     your `beans.xml`.
 
+[NOTE]
+This module has been *experimentally* shown to be compatible
```

Comment:
What additional validation would we need to do to drop the '*experimentally*' here?  Spring 5 is the GA version, if we can claim to support it, we should.

Reply:
don't know. It seems to me that if its basic usage works and everything compiles, it probably just works, but I'm not the expert on it so I'm not gonna state it as fact without a thorough test. I guess I would set up a jdbi-spring5 that _only_ contains tests and doublecheck tx handling and stuff there?

Reply:
On that note btw, what's up with #1252?

Reply:
I'm not sure exactly.  Nobody in the core project understands Spring transaction management (or wants to use it) and it's a lot of code to merge that we don't really understand.  And it's not currently quite ready to ship anyway.  So it's stuck in limbo...

Reply:
Hmm, yeah, I see what you mean. The first half isn't so bad, but then you get to the real spring tx integration part...